--- minizip-ng-4.0.10/CMakeLists.txt.orig	2025-05-05 03:36:06.000000000 +0200
+++ minizip-ng-4.0.10/CMakeLists.txt	2025-11-30 18:46:29.869161479 +0100
@@ -724,12 +724,6 @@ if(NOT SKIP_INSTALL_LIBRARIES AND NOT SK
     write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/${MINIZIP_TARGET}-config-version.cmake
         COMPATIBILITY SameMajorVersion)
     set(MINIZIP_CONFIG_CONTENT "@PACKAGE_INIT@\n")
-    if(MINIZIP_DEP_PKG)
-        string(APPEND MINIZIP_CONFIG_CONTENT "include(CMakeFindDependencyMacro)\n")
-        foreach(pkg_name ${MINIZIP_DEP_PKG})
-            string(APPEND MINIZIP_CONFIG_CONTENT "find_dependency(${pkg_name})\n")
-        endforeach()
-    endif()
     string(APPEND MINIZIP_CONFIG_CONTENT "include(\"\${CMAKE_CURRENT_LIST_DIR}/${MINIZIP_TARGET}.cmake\")")
     file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/minizip-config.cmake.in ${MINIZIP_CONFIG_CONTENT})
 
